content/02.main-text.md

```diff
@@ -210,6 +220,8 @@ For example, figures can be specified using versioned URLs that refer to the cod
 In addition, manuscripts can be templated, so that numerical values or tables are inserted directly from the repository that created them.
 An [example repository](https://github.com/manubot/rootstock) demonstrates Manubot's features and serves as a template for users to write their own manuscript with Manubot.
 
+### Existing Manubot manuscripts
```

Comment:
Should we call this `Exiting Manubot usage` or `Manubot examples`?  I'm not opposed to the current header if you prefer to keep it.